File: Factorial.py
"""
Problem Link:https://www.codechef.com/LRNDSA01/problems/FCTRL
@author: subho57
"""
t=int(input())
while(t):
    count=0
    n=int(input())
    while n>=5:
        n//=5
        count+=n
    print(count)
    t-=1

File: Factorial.c
/*
Problem Link : https://www.codechef.com/LRNDSA01/problems/FCTRL
@author: subho57
*/
#include <stdio.h>
int main(void){
    int t;
    scanf("%d", &t);
    while(t--){
        int n;
        scanf("%d", &n);
        int count=0;
        while(n>=5){
            n/=5;
            count+=n;
        }
        printf("%d\n", count);
    }
}
